README.md

```diff
@@ -23,6 +23,16 @@ The infrastructure, deployment, and operational parts of this project are identi
 
 > _Note_: When packaging/deploying the serverless application you'll not only need the normal `STAGE`, etc. environment variables in that guide but will also need a `API_KEY_SECRET` value.
 
+## Dependencies
+
+This project consumes `@formidablelabs/dogs` from [GitHub Package Registry](https://github.com/features/packages) which requires the following modification to your `.npmrc` file.
+
+```sh
+# GITHUB_PAT is a personal access token - https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token
```

Comment:
Let's include a bit more discussion about creating the token. In particular, minimizing misuse risk by scoping to `read:packages`